README.md

```diff
@@ -37,6 +37,7 @@ http://jkusa.github.io/ember-cli-clipboard
 * `clipboardText` - string value to be copied
 * `clipboardTarget` - selector string of element from which to copy text
 * `clipboardAction` - string value of operation: `copy` or `cut` (default is copy)
+* `useClipboardNode` - boolean to scope element listener to copy-button component (default false)
```

Comment:
Yeah that seems simpler - `delegateClickEvent: true` is the default; otherwise if you want to scope it to the copy-button component, `delegateClickEvent: false`.